content/embeds/backup-locations.md

```diff
@@ -35,12 +35,14 @@ For example: `sftp://username:password@10.1.1.1/home/backups/`
 
 ### AWS S3
 
-Before you choose to backup to Amazon S3, make sure that you have:
+Before you choose to backup to Amazon AWS S3, make sure that you have:
 
 - Storage location path in the format: `s3://bucketname/path/`
 - Access key ID
 - Secret access key
 
+To connect to a locally-hosted AWS S3 location, run: `rladmin cluster config s3_url <hostname>`
```

Comment:
@guyco - What do you think?
Please refer to two issues:
1. The name “locally-hosted AWS S3 location” - is it accurate name?
2. Do the customer need to handle any certs in order for this command to work?

Reply:
@guyco-redis - Thanks.
@bmansheim - Can you please update the PR accordingly?

Reply:
Done